### ISSUE TITLE
[Data Mapper] Failed to render the Statement editor for values with conditional expressions

### ISSUE THREAD

**Issue by madushajg** [wso2/ballerina-plugin-vscode#439]
**Description:**
$Subject
![conditional-expr-error](https://user-images.githubusercontent.com/26219651/222677366-af3acd74-93c7-46a9-8e48-cc5a9e6f8218.gif)

**Steps to reproduce:**
Source:
```ballerina
type Person record {
    string id;
    string firstName;
    string lastName;
    int age;
    string country;
};

type Student record {
    string visaType;
};

boolean isForeign = true;

function transform(Person person) returns Student => {
        visaType: isForeign ? "visaType" : ""
    };
```

**Affected Versions:**
Ballerina 2201.3.2
Plugin v3.3.11

**Comment by madushajg** [comment]
This is causing due to: https://github.com/ballerina-platform/ballerina-lang/issues/39799

**Comment by madushajg** [comment]
Update: the fix will be available from Ballerina `2201.5.0`, putting the issue `on-hold` until that.

**Comment by madushajg** [comment]
Tested with Ballerina `2201.5.0` and the issue has been fixed. Hence closing the issue.
